src/index.js

```diff
@@ -1 +1,11 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import createGame from './modules/createGame.js';
+import showScore from './modules/showScore.js';
+/* eslint-disable */
```

Comment:
- [ ] [OPTIONAL] The error is probably because of this windows - global event. Kindly bear in mind that it is not always professional to avoid best practices like this, as it is similar to the workflows used in real life working environments :pleading_face: